Makefile

```diff
@@ -2,7 +2,7 @@
 
 PYTHON := $(shell which python)
 CYTHON := $(shell which cython)
-NOSETESTS := $(shell which nosetests)
+NOSETESTS := $(shell which pytest)
```

Comment:
Hi @henryiii, here you meant
`PYTESTS :=` …

Reply:
Yes I did!